client/pages/home.jsx

```diff
@@ -8,10 +8,15 @@ const App = () => {
   const [imageLinks, setImageLinks] = useState([]);
   const [title, setTitle] = useState('Results');
   const [imageFlag, setImageFlag] = useState(false);
+  const [isFavorites, setFavorites] = useState(false);
 
   useEffect(() => {
-    if ((window.location.search + '').indexOf('avorit') > 0) {
-      favorites();
+
+    const f = localStorage.getItem('favorites');
+    if ((window.location.search + '').indexOf('avorit') > 0 || f === 'true') {
```

Comment:
Instead of using both of these, you should either use localStorage or the url, but not both